src/Utilities.mss

```diff
@@ -432,12 +432,14 @@ function AddControlEventAttributes (bobj, element) {
         }
     }
 
-    if (bobj.Color != 0)
+    if (bobj.Type != 'Text')
```

Comment:
I think that's not the check we need here. For example, `<harm>` is allowed to have color. This would prevent export of color e.g. for figured bass.

Reply:
In MEI 5 `color` is available for all control elements, see https://github.com/music-encoding/music-encoding/pull/1457.